src/main/java/org/folio/service/export/storage/ExportStorageFactory.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.service.export.storage;
+
+import java.lang.invoke.MethodHandles;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExportStorageFactory {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+
+  public ExportStorageService getExportStorageImplementation(String type) {
```

Comment:
(minor) Idea for refactoring - in case if we start to create AWSStorageServiceImpl as spring bean instead of using constructor then you can use initialize Map with StorageService instances as values and with string types as keys. In this "switch" can be removed

Reply:
You have to excuse that my spring knowledge  is close to none. Can you point me to any reference ?

Reply:
Let's keep it simple, switch or if-else is enough

Reply:
yea, I already changed it just to return the AWS imlementation